CRM/Civicase/APIHelpers/CaseDetails.php

```diff
@@ -361,4 +363,63 @@ private static function prepareParamsForFiltering(array &$params, $paramName) {
     }
   }
 
+  /**
+   * Handles filtering of case custom fields with multiple options.
+   *
+   * @param CRM_Utils_SQL_Select $sql
+   *   Reference to the SQL object.
+   * @param array $params
+   *   The list of params as provided by the action.
+   */
+  private static function handleCustomFieldFiltering(CRM_Utils_SQL_Select $sql, array &$params) {
```

Comment:
Is it possible to move the `handleCustomFieldFiltering` and `joinCustomFieldQuery` functions to a different helper and reference them on this file? I'm worried that this file is growing too large.